services/movie_session.py

```diff
@@ -20,15 +19,17 @@ def get_movies_sessions(session_date: str = None) -> QuerySet:
     return queryset
 
 
-def get_movie_session_by_id(movie_session_id: int) -> MovieSession:
-    return MovieSession.objects.get(id=movie_session_id)
+def get_taken_seats(movie_session_id: int) -> list[dict]:
+    tickets = Ticket.objects.filter(
+        movie_session_id=movie_session_id).values("row", "seat")
+    return list(tickets)
 
 
 def update_movie_session(
     session_id: int,
     show_time: str = None,
     movie_id: int = None,
-    cinema_hall_id: int = None,
+    cinema_hall_id: int = None
 ) -> None:
     movie_session = MovieSession.objects.get(id=session_id)
```

Comment:
Consider handling the `DoesNotExist` exception that can be raised by `MovieSession.objects.get(id=session_id)`. This will prevent runtime errors if the session ID is not found.